src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -370,15 +370,19 @@ const AccountClosure = (() => {
             } else {
                 $submit_loading.setVisibility(0);
                 $closure_container.setVisibility(0);
+                $('.client_logged_in, #client-logged-in, #topbar-msg').setVisibility(0);
                 $success_msg.setVisibility(1);
+                $('.client_logged_out').setVisibility(1);
+                $('#topbar').addClass('primary-bg-color-dark');
+                $('#menu-top').removeClass('smaller-font top-nav-menu');
+                $('#topbar').removeClass('secondary-bg-color');
```

Comment:
why are we faking the logout? 🤔

Reply:
Initially I didn't. know another way since the webpages are hardwired. Thus, if we would actually log the user out to show the logged out menu, this user wont see the message anymore, since that page is only available for logged in users.

What I did now is using the already available closingAccount session to display the logged out menu when the user closes his/her account.